webapp/components/PasswordReset/Request.spec.js

```diff
@@ -18,7 +18,7 @@ describe('Request', () => {
         success: jest.fn(),
         error: jest.fn(),
       },
-      $t: jest.fn(),
+      $t: jest.fn(t => t),
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-09T14:02:11Z" title="Wednesday, October 9th 2019, 4:02:11 pm +02:00">Oct 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -18,7 +18,7 @@ describe('Request', () => {
         success: jest.fn(),
         error: jest.fn(),
       },
-      $t: jest.fn(),
+      $t: jest.fn(t => t),
```


we just ran into this issue today, on `ReportModal.spec.js`... would love to refactor all the `$t: jest.fn()` like this

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-10T09:06:51Z" title="Thursday, October 10th 2019, 11:06:51 am +02:00">Oct 10, 2019</time>_

---


I know.. we learn everyday. From now on I will always check if `wrapper.contains('some.translation.key')` instead of `expect(mocks.$t).toHaveBeenCalled('some.translation.key')`